ParkingZoneApp/Areas/Identity/Pages/Account/Manage/Index.cshtml

```diff
@@ -12,12 +12,12 @@
         <form id="profile-form" method="post">
             <div asp-validation-summary="ModelOnly" class="text-danger" role="alert"></div>
             <div class="form-floating mb-3">
-                <input asp-for="Username" class="form-control" placeholder="Please choose your username." disabled />
-                <label asp-for="Username" class="form-label"></label>
+                <input asp-for="Input.Name" class="form-control"/>
+                <label asp-for="Input.Name" class="form-label">Enter your name</label>
```

Comment:
Where is the UserName that shows Email? Why in Index are not you showing it

Reply:
![ParkingZonePhoto](https://github.com/dilmurod-rozikov/ParkingZone/assets/100861214/5b3082e8-d4b3-4417-9ffb-e374e2509354)

Reply:
![image](https://github.com/dilmurod-rozikov/ParkingZone/assets/141717735/61d9d679-f3ab-4239-a476-6856c67c237b)
So name and username both two different thing and username is email right?

Reply:
yeah right